api/testworkflows/v1/service_types.go

```diff
@@ -38,6 +38,9 @@ type IndependentServiceSpec struct {
 
 	// Probe to check if the service has started correctly
 	ReadinessProbe *corev1.Probe `json:"readinessProbe,omitempty" expr:"force"`
+
+	// list of accompanying permanent volume claims
+	Pvcs map[string]TestWorkflowPvcConfig `json:"pvcs,omitempty" expr:"template,include"`
```

Comment:
Maybe it would be simpler to just use `map[string]corev1.PersistentVolumeClaimSpec`?

Reply:
yes, we can, should not be  a problem